documentation/release-notes/19_3.md

```diff
@@ -0,0 +1,93 @@
+# run_statistics 
+
+---
+
+###Total number of reduction jobs for each instrument
+
+* GEM: 5688
+* HRPD: 986
+* MAPS: 1176
+* MARI: 545
+* MUSR: 1342
+* OSIRIS: 1185
+* POLARIS: 2810
+* POLREF: 650
+* WISH: 748
+
+*Total*
+15130
+
+---
+
+###Total number of unique runs for each instrument
+* ENGINX: 5688
+* GEM: 981
+* MAPS: 1074
+* MARI: 326
+* MUSR: 1340
+* OSIRIS: 1183
+* POLARIS,: 2798
+* POLREF: 646
+* WISH: 681
+
+*Total*
+15017
+
+---
+
+###Total number of Failed runs for each instrument
+
+* ENGINX: 40
+* MAPS: 23
+* MARI: 57
+* POLARIS: 1
+* WISH: 11
+
+*Total*
+132
+
+---
+
+###Resolved failures for runs
+* GEM: 4
+* WISH: 1
+* OSIRIS: 2
+* POLARIS: 9
+* MUSR: 2
+* POLREF: 3
+* MARI: 47
+* MAPS: 7
+
+*Total*
+75
+
+---
+#Monitoring
+###Find all failed runs since date
+
+* ENGINX: 40
+* WISH: 70
+* MARI: 201
+* MAPS: 99
+
+*Total*
+410
+
+---
+
+###Find all failed runs that did not complete since date
+
+* GEM: 4
+* WISH: 12
+* OSIRIS: 2
+* POLARIS: 9
+* MUSR: 2
+* MARI: 72
+* MAPS: 24
+* POLREF: 3
+
+*Total*
+128
+
+---
+
```

Comment:
At the end of the file can we have 2 extra calculations:
Job success: (Reduction job - Failed Jobs) / Reduction Jobs (%)
Run Success: (Reduction runs - Failed Runs + Successful re-runs) / Reduction runs (%)